js_modules/dagit/packages/app/src/NUX/CommunityNux.tsx

```diff
@@ -16,17 +17,25 @@ import React from 'react';
 import isEmail from 'validator/lib/isEmail';
 
 export const CommunityNux = () => {
-  const [didDismissCommunityNux, dismissCommunityNux] = useStateWithStorage(
+  const [didDismissCommunityNux, dissmissInBrowser] = useStateWithStorage(
     'communityNux',
     (data) => data,
   );
-  if (didDismissCommunityNux) {
+  const {data, loading} = useQuery(GET_SHOULD_SHOW_NUX_QUERY);
+  const [dismissOnServer] = useMutation(SET_NUX_SEEN_MUTATION);
```

Comment:
These should be updated to use type annotations again. The lint rule might not be working right now.

Reply:
Interestingly... the codegen didn't generate any types in the folder... Could it be because there are no arguments or returned data from the mutation? And because the query just returns a primitive directly rather than an object?